finance/fraud_detection/src/main/resources/schema.tql

```diff
@@ -0,0 +1,206 @@
+#
+# Copyright (C) 2022 Vaticle
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# dataset without banks
+# https://www.kaggle.com/datasets/kartik2112/fraud-detection?select=fraudTrain.csv
+
+ define
+
+name sub attribute,
+    value string;
+
+first_name sub attribute,
+    value string;
+
+last_name sub attribute,
+    value string;
+
+job sub attribute,
+    value string;
+
+gender sub attribute,
+    value string,
+    regex "^(F|M|O)$";
+
+company_type sub attribute,
+    value string;
+
+timestamp sub attribute,
+    value datetime,
+    plays customer_relationship:timestamp,
+    plays same_place:timestamp;
+
+date_of_birth sub attribute,
+    value datetime;
+
+card_number sub attribute,
+    value long;
+
+amount sub attribute,
+    value double;
+
+is_safe sub attribute,
+    value boolean;
+
+transaction_number sub attribute,
+    value string;
+
+street sub attribute,
+    value string;
+
+city sub attribute,
+    value string;
+
+state sub attribute,
+    value string;
+
+zip sub attribute,
+    value long;
+
+longitude sub attribute,
+    value double;
+
+latitude sub attribute,
+    value double;
+
+Geo_coordinate sub entity,
```

Comment:
Hmm. Our general TypeQL code style is lowercase (kebab-case) for all TypeQL schema types. Why are we using an initial capital letter for entities here?